next.config.js

```diff
@@ -198,18 +198,6 @@ module.exports = withPlugins([[withBundleAnalyzer]], {
       },
     ];
   },
-  async rewrites() {
-    return {
-      fallback: [
-        // These rewrites are checked after both pages/public files
-        // and dynamic routes are checked
-        {
-          source: '/profile/:slug*',
```

Comment:
Let's hope not so much links of type https://www1.plant-for-the-planet.org/profile/norbert-schuler exists any more anywhere in the web or within our own application :-)